transcom/mymove
# INT B-20110 refactor CustomerInfoPanel to use Formik

## [Agility ticket](https://www13.v1host.com/USTRANSCOM38/assetdetail.v1?number=B-20110)

## Summary

We are not wanting to use `redux-form` and instead use Formik due to library support. This backlog item addresses the `CustomerInfoPanel` component and its use of `redux-form`.

Luckily, this component isn't being used ANYWHERE in the code base and was tech debt. Removing the file solves that problem real quick. SAYONARA SUCKA.

### How to test

1. In VSCode, search for `redux-form`
2. You should no longer see it being used in the `CustomerInfoPanel` component
3. Confirm that `CustomerInfoPanel` no longer exists in the code base and that it isn't being used anywhere else
